geonode_mapstore_client/client/js/components/home/FilterForm.jsx

```diff
@@ -96,79 +96,89 @@ function FilterForm({
+                                            multi
+                                            placeholder={placeholderId}
+                                            onChange={(selected) => {
+                                                setValues({
+                                                    ...state.current.values,
+                                                    [filterKey]: selected.map(({ value }) => value)
+                                                });
+                                            }}
+                                            { ...optionsProp }
+                                        />
+                                        {description &&
                                         <Form.Text className="text-muted">
                                             {description}
                                         </Form.Text>}
-                                </Form.Group>
-                            );
-                        })}
-                    </Col>}
-                    <Col style={{
-                        height: leftColumnNode.current?.clientHeight || 400
-                    }}>
-                        <FilterByExtent
-                            id={id}
-                            extent={values.extent}
-                            queryExtent={query.extent}
-                            layers={extentProps?.layers}
-                            vectorLayerStyle={extentProps?.style}
-                            onChange={({extent}) =>
-                                setValues({
-                                    ...values,
-                                    extent
-                                })
-                            }
-                        />
-                    </Col>
-                </Form.Row>
-            </Form>
+                                    </Form.Group>
+                                );
+                            })}
+
+                            <FilterByExtent
+                                id={id}
+                                extent={values.extent}
+                                queryExtent={query.extent}
+                                layers={extentProps?.layers}
+                                vectorLayerStyle={extentProps?.style}
+                                onChange={({extent}) =>
+                                    setValues({
+                                        ...values,
+                                        extent
+                                    })
+                                }
+                            />
+
+                        </Col>
+                    </Form.Row>
+                </Form>
+            </div>
             <div className="gn-filter-form-footer">
                 <Button
-                    variant="default"
+                    variant="primary"
```

Comment:
If we decide to use the variant primary we should also update this class in the custom style to support css native variables. I'll open a separated issue for this